web-registry/src/hooks/useEcocreditQuery.ts

```diff
@@ -155,6 +161,9 @@ export default function useEcocreditQuery<T extends EcocreditQueryResponse>({
       case 'creditTypes':
         response = creditTypes(client, params);
         break;
+      case 'params':
```

Comment:
Adding params is not necessary anymore for this PR but I kept the change as it can be useful later on.